okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -149,6 +149,8 @@ open class OkHttpClient internal constructor(
   private val readTimeout: Int = builder.readTimeout
   private val writeTimeout: Int = builder.writeTimeout
   private val pingInterval: Int = builder.pingInterval
+  private val webSocketCompression: Boolean = builder.webSocketCompression
```

Comment:
Is it possible to have a number of compression algorithms? In such case it would be advisable to have `enum` here, like:

```kotlin
enum class Compression { PermessageDeflate, None }
```

Reply:
AFAIK, "permessage-deflate" is the only widely used. Don't see a reason to overcomplicate ATM.